release-notes-2.4.0.md

```diff
@@ -4,5 +4,6 @@
 ${version-number}
 
 #### New Features
+- SCMOD-13948: dirmngr package is required when using latest opensuse-leap 15.3.
```

Comment:
Is this really accurate? What exactly is the dirmngr package required for? It is something to do with the keyserver so add a little more detail on why we need it. Also state that it was removed from the OS packages as leap 15.2 had it pre-installed.